src/components/specimen/components/contentBlocks/Identifications.tsx

```diff
@@ -0,0 +1,100 @@
+/* Import Dependencies */
+import { isEmpty, cloneDeep } from 'lodash';
+import classNames from 'classnames';
+import { Card, Row, Col } from 'react-bootstrap';
+
+/* Import Store */
+import { useAppSelector } from 'app/hooks';
+import { getSpecimen } from 'redux/specimen/SpecimenSlice';
+
+/* Import Types */
+import { Dict } from 'app/Types';
+
+/* Import Components */
+import PropertiesTable from './PropertiesTable';
+
+
+/* Props Typing */
+interface Props {
+    ShowWithAnnotations: Function
+}
+
+
+const Identifications = (props: Props) => {
+    const { ShowWithAnnotations } = props;
+
+    /* Base variables */
+    const specimen = useAppSelector(getSpecimen);
+    let identifications: {
+        properties: Dict,
+        [taxonIdentification: string]: Dict
+    }[] = [];
+
+    /* Craft identifications object to iterate over */
+    specimen.digitalSpecimen['dwc:identification']?.forEach((identification, index) => {
```

Comment:
So you basically flatten the object so that it fits easier into the table. Wonder if we can have collapsable sections in the table to separate the different identifications. Maybe something to discuss/brainstorm

Reply:
Yeah exactly, also tried to explain this in the PR of the occurrences tab. Yes that is a good idea to make them collapsable, will add that task to the board